packages/base/src/commands.ts

```diff
@@ -285,6 +290,177 @@ export function addCommands(
     ...icons.get(CommandIDs.newVectorTileEntry)
   });
 
+  commands.addCommand(CommandIDs.buffer, {
+    label: trans.__('Buffer'),
+    isEnabled: () => {
```

Comment:
You'd need to disable that command if the layer does not support it:

See https://github.com/geojupyter/jupytergis/pull/510/files#diff-d081b9801255d01815fd2c61a5863deab9e4d0f769d10f165d4dc6f5d0cf082eR71